src/test/java/com/epam/novostroinyi/ui/junit/BaseJUnitTest.java

```diff
@@ -1,31 +1,45 @@
 package com.epam.novostroinyi.ui.junit;
 
-import static com.epam.novostroinyi.core.config.ConfigUtils.setBrowser;
+import static com.epam.novostroinyi.core.constant.LaunchesConstants.LAUNCH_EXPORT_DIR_PATH;
 import static com.epam.novostroinyi.core.util.WebDriverUtils.openUrl;
+import static com.epam.novostroinyi.core.util.WebDriverUtils.setBrowser;
 
+import com.codeborne.selenide.Configuration;
 import com.epam.novostroinyi.core.config.Property;
+import com.epam.novostroinyi.core.constant.Browser;
 import com.epam.novostroinyi.core.util.WebDriverUtils;
-import com.epam.novostroinyi.step.LoginPageSteps;
+import com.epam.novostroinyi.ui.BaseUiSteps;
+import com.epam.novostroinyi.ui.step.LoginPageSteps;
+import lombok.SneakyThrows;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.runner.RunWith;
 
-public class BaseJUnitTest {
+@RunWith(TestRunner.class)
+public class BaseJUnitTest implements BaseUiSteps {
```

Comment:
BaseJunitTest and BaseUiSteps should be from different layers, that's why BaseTest cannot implement BaseStep